src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -18,7 +18,8 @@ public Manufacturer create(Manufacturer manufacturer) {
 
     @Override
     public Manufacturer get(Long id) {
-        return manufacturerDao.get(id).get();
+        return manufacturerDao.get(id).orElseThrow(() ->
+                new RuntimeException("Useless method get."));
```

Comment:
Let's change the message to "Couldn't get a manufacturer from DAO by id = " + id

Reply:
Added